### PR TITLE
Feature/dcmfoss 47- Material Demand Overview/Values

### PR DESCRIPTION
**Pull request by Ruskyy**
## Description
Material Demands now have a UI for declaring demand values.
- If I want to edit DemandAmounts for a specific Material, I have to open the details view for that material.

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [X] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [X] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

**Review by carslen — CHANGES_REQUESTED** [review]
In your PR description you checked the checkbox for DEPENDENCIES are up to date, but:

- there have been changes to `demand-capacity-mgmt-frontend/package-json.lock`, so dependencies changed,
- file `demand-capacity-mgmt-frontend/DENEDENCIES_FRONTEND` did not get updated

It is essential to have your dependencies documented using the `DEPENDENCIES` file (or `DEPENDENCIES_FRONTEND` in this case).

Please update the `DEPENDECIES_FRONTEND`file in repositories root directory.

**Comment by carslen** [comment]
Pleas handle the error menioned in the `build-frontend / build-and-analyde-frontend`check, details can be [found here](https://github.com/eclipse-tractusx/demand-capacity-mgmt/actions/runs/6161886242/job/16722127901?pr=15).

File: demand-capacity-mgmt-backend/src/main/resources/application.yml [hunk 1]
```diff
@@ -25,8 +25,8 @@ spring:
     show-sql: true
   datasource:
     url: jdbc:postgresql://${DCM_DATASOURCE_HOST:localhost:5432}/${DCM_DATASOURCE_NAME:dcm}
-    username: ${DCM_DATASOURCE_USER:postgres}
-    password: ${DCM_DATASOURCE_PASS:postgres}
+    username: ${DCM_DATASOURCE_USER:dcm}
+    password: ${DCM_DATASOURCE_PASS:dcm}
```
Comment (carslen) [inline demand-capacity-mgmt-backend/src/main/resources/application.yml:29]: Please do not hard code username and password but make it configurable in a way that you can set/control it in your product helm chart `values.yaml` file.

**Comment by Ruskyy** [comment]
@carslen Issues should be fixed. 
The dependencies file was done, but i had forgotten to commit.

**Comment by carslen** [comment]
> @carslen Issues should be fixed. The dependencies file was done, but i had forgotten to commit.

Perfect!

**Review by carslen — APPROVED** [review]

**Review by Bailonis — APPROVED** [review]
Was talking with @Ruskyy and everything seems good to me

**Pull request merged**